bezier-rs/lib/src/lib.rs

```diff
@@ -70,18 +70,61 @@ impl Bezier {
 		}
 	}
 
+	// TODO: Maybe move to a utils file
+	fn compute_abc(start_point: DVec2, point_on_curve: DVec2, end_point: DVec2, t_term: f64, one_minus_t_term: f64) -> [DVec2; 3] {
+		let point_c_ratio = one_minus_t_term / (t_term + one_minus_t_term);
+		let c = point_c_ratio * start_point + (1. - point_c_ratio) * end_point;
+		let ab_bc_ratio = (t_term + one_minus_t_term - 1.).abs() / (t_term + one_minus_t_term);
+		let a = point_on_curve + (point_on_curve - c) / ab_bc_ratio;
+		[a, point_on_curve, c]
```

Comment:
I think you could return a tuple `(a, points_on_curve, c)` here instead, and this would allow you to use destructuring syntax at the callsites (e.g. `let a, b, c = compute_abc(...)`), which is a bit cleaner

Reply:
You can still destructure with array syntax. Since these are sequential (i.e. the ordering is a relevant factor), an array seems slightly more appropriate, but the distinction is both subtle and opinionated. Either way is fine as far as I'm concerned.